server/controllers/incidentRecord.js

```diff
@@ -1,37 +1,36 @@
 import { incidentsDB } from "../dummyDB";
 import { Helpers } from "../helpers/predefinedMethods";
 
-const red_flag_string = "red-flag";
-
-let specifiedRedFlagRecordId = "";
-let allRedFlagsRecords = "";
-let specificRedFlag = "";
 
 export class Incidents {
-	/**filter Incident Record(s)
-  * @param  { object } req - Contains the body of the request.
-  * @param { object } res - Contains the returned response.
-  */
+	/**
+   * filter Incident Record(s)
+   * @param  { object } req - Contains the body of the request.
+   * @param { object } res - Contains the returned response.
+   * @returns { Array } returns an array of numbers and an array of array of object
+   */
 	static filterRecords (req, res, incident_record) {
-		specifiedRedFlagRecordId = parseInt(req.params.id, 10);
-		allRedFlagsRecords = incidentsDB.filter((redFlag) => redFlag.type === incident_record);
-		specificRedFlag = allRedFlagsRecords.filter((redFlagId) => redFlagId.id === specifiedRedFlagRecordId);
+
+		const specifiedRedFlagRecordId = parseInt(req.params.id, 10);
+		const allRedFlagsRecords = incidentsDB.filter((redFlag) => redFlag.type === incident_record);
+		const specificRedFlag = allRedFlagsRecords.filter((redFlagId) => redFlagId.id === specifiedRedFlagRecordId);
 
 		return [specifiedRedFlagRecordId, allRedFlagsRecords, specificRedFlag];
 	}
 
 
-	/**Creates a red-flag or intervention record
-  * @param  { object } req - Contains the body of the request.
-  * @param { object } res - Contains the returned response.
-  */
+	/**
+   * Creates a red-flag or intervention record
+   * @param  { object } req - Contains the body of the request.
+   * @param { object } res - Contains the returned response.
```

Comment:
```suggestion
   * @param {object} res - Contains the returned response.
```
Please remove the spaces around the param type